alf/networks/projection_networks.py

```diff
@@ -765,3 +765,68 @@ def forward(self, inputs, state=()):
             upper_bound=self._action_high)
 
         return dist, state
+
+
+@alf.configurable
+class OnehotCategoricalProjectionNetwork(Network):
```

Comment:
This seems more appropriately a transformation/wrapper on top of the existing CategoricalProjectionNetwork: 
``` python
dist = OnehotCategorical(Categorical)
```

Reply:
> This seems more appropriately a transformation/wrapper on top of the existing CategoricalProjectionNetwork:
> dist = OnehotCategorical(Categorical)
> 

The suggested one is a possible alternative approach, but my current thought is that it is unnecessary as it creates another layer of complication for the user.